scripts/docker-entrypoint.sh

```diff
@@ -8,7 +8,7 @@ else
 fi
 
 if ${BINARY} help "$1" 2>&1 | grep -q "shipitron $1"; then
-  set -- gosu shipitron ${BINARY} "$@"
+  set -- su-exec shipitron ${BINARY} "$@"
```

Comment:
So we're making the switch to `su-exec` eh?

Reply:
Yep.